integration/run_test.go

```diff
@@ -155,20 +126,27 @@ func createCRDExamples(t *testing.T) {
 	}
 }
 
-func deleteFailedDeployments() {
-	deleteWebhookCmd := exec.Command("kubectl", "delete", "validatingwebhookconfiguration", "--all")
-	integration_util.RunCmdOut(deleteWebhookCmd)
-	helmCmd := exec.Command("sh", "-c",
-		"helm ls --short | xargs -L1 helm delete")
-	helmCmd.Dir = "../artifacts"
-	integration_util.RunCmdOut(helmCmd)
-}
+func createGACSecret(t *testing.T, ns *v1.Namespace) {
+	crdCmd := exec.Command("gsutil", "cp", "gs://kritis-test-files/gac.json", "/tmp")
```

Comment:
Ignore if this is impossible to handle otherwise, but I worry that this might be causing the tests to more non-hermetic than necessary, by introducing a dependency on an external data source that might change outside of the test.

Is it possible to generate the secret on the fly?

Reply:
since the tests run in a different namespace every time we need to generate the secret during runtime, but I switched this from copying the creds from a GCS bucket to copying them directly into kokoro first.